maps/src/test/java/org/odk/collect/maps/layers/OfflineMapLayersPickerTest.kt

```diff
@@ -37,24 +44,36 @@ import org.odk.collect.webpage.ExternalWebPageHelper
 
 @RunWith(AndroidJUnit4::class)
 class OfflineMapLayersPickerTest {
-    private val referenceLayerRepository = mock<ReferenceLayerRepository>().also {
-        whenever(it.getAll()).thenReturn(emptyList())
-    }
+    private val referenceLayerRepository = mock<ReferenceLayerRepository>()
     private val scheduler = FakeScheduler()
     private val settingsProvider = InMemSettingsProvider()
     private val externalWebPageHelper = mock<ExternalWebPageHelper>()
 
+    private val uris = mutableListOf<Uri>()
+    private val testRegistry = object : ActivityResultRegistry() {
```

Comment:
I'd prefer if this was pulled out to a concrete stub that you can set a `uris` field on directly. That would feel a little clearer and contains the `uris` so you know where they're being used quickly.

Reply:
Done.